src/main/java/Duke.java

```diff
@@ -8,21 +8,39 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
 
-        String line = "_________________________________\n";
+        String line = "____________________________________________________\n";
         System.out.println("Hello from\n" + logo);
         System.out.print(line + "Hello! I'm Duke\n" + "What can I do for you?\n" + line);
         String[] list = new String[100];
```

Comment:
Naming of  ' String[ ]  list  ' is quite vague, maybe if possible to provide a more useful name for readability? e.g. ' String [ ] taskList '